src/buildstream/buildelement.py

```diff
@@ -286,10 +306,20 @@ def configure_sandbox(self, sandbox):
         sandbox.set_work_directory(command_dir)
 
         # Setup environment
-        sandbox.set_environment(self.get_environment())
+        env = self.get_environment()
 
-    def stage(self, sandbox):
```

Comment:
nit: removing the empty line after `def stage(self, sandbox):` results in a confusing diff.